components/channel_view/index.js

```diff
@@ -24,10 +25,14 @@ const getDeactivatedChannel = createSelector(
 function mapStateToProps(state) {
     const channelId = state.entities.channels.currentChannelId;
 
+    const config = state.entities.general.config;
```

Comment:
non-blocking but there is a selector for `getConfig` and another for `getLicense` if you want to change those

Reply:
Aha, yes, I'll switch to using that.